app/models/event.rb

```diff
@@ -10,4 +11,8 @@ class Event < ApplicationRecord
             presence: true
 
   scope :upcoming, -> { where('date >= ?', Time.zone.today) }
+  scope :requiring_notification, -> do
+    where('date = ?', Date.current - NOTIFICATION_PERIOD)
+      .includes(presentations: [:person])
```

Comment:
The `[]` around person here is not needed.

In this case I wouldn't preload the associations here (unless the scope itself needed it for some reason). you take this scope from being 1 query to 3, and you might not always want the associated records.

Better to leave the `includes` call to the person who will actually be using the scope